src/pages/fr/guides/troubleshooting.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Dépannage
+description: Besoin d'aide ? Bloqué sur quelque chose ? Ce guide est fait pour vous.
+---
+
+Astro fournit plusieurs outils différents pour vous aider à dépanner et à déboguer votre code.
+
+## Erreurs courantes
+
+Voici une liste de quelques messages d'erreurs courants que vous pourriez rencontrer dans votre terminal, ce qu'ils signifient et ce que vous pouvez faire pour les corriger.
+
+### Transform failed with X error
+
+Ce message apparaît souvent à cause d'une limitation actuelle dans Astro qui exige que vos appels `import` et `export` soient au début de votre fichier `.astro`.
+
+**Solution**: Écrivez vos imports et exports au début de votre Script de composant.
+
+**Statut**: Limitation actuelle; Un fix est en cours de développement.
+
+💡 Vous n'êtes pas certain que cela va résoudre votre problème ? Vérifiez si quelqu'un d'autre a déjà [signalé ce type d'erreur](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Transform+failed+with+*+error) !
+
+### Cannot use import statement outside a module
+
+Dans les composants Astro, les balises `<script>` sont hoistées et chargées comme des [modules JavaScript](https://developer.mozilla.org/fr/docs/Web/JavaScript/Guide/Modules) par défaut. Si vous avez ajouté la directive [`is:inline`](/fr/reference/directives-reference/#isinline) ou n'importe quelle autre attribut dans votre balise, ce comportement par défaut est désactivé.
```

Comment:
Dans les composants Astro, les balises `<script>` sont hoistées et chargées comme des [modules JavaScript](https://developer.mozilla.org/fr/docs/Web/JavaScript/Guide/Modules) par défaut. Si vous avez ajouté la directive [`is:inline`](/fr/reference/directives-reference/#isinline) ou n'importe quel autre attribut dans votre balise, ce comportement par défaut est désactivé.